advanced.md

```diff
@@ -167,4 +167,112 @@ public void toggleAllDigitalPins( bool setPinsHigh )
 		firmata.sendDigitalPort( 1, 0 );	//all pins low
```

Comment:
Thank you for the offline explanation. In this case, I see what you were trying to do and approve.